manifest_staging/charts/secrets-store-csi-driver/templates/secrets-store-csi-driver-windows.yaml

```diff
@@ -64,6 +64,9 @@ spec:
             - "--rotation-poll-interval={{ .Values.rotationPollInterval }}"
             {{- end }}
             - "--metrics-addr={{ .Values.windows.metricsAddr }}"
+            {{- if .Values.windows.prometheusPort }}
+            - "--prometheus-port={{ .Values.windows.prometheusPort }}"
```

Comment:
To be consistent, let's also explicitly add `--prometheus-port=` to the corresponding deployment yaml in /deploy/

Reply:
Done!